fuel-core/src/database/validator_set_diffs.rs

```diff
@@ -0,0 +1,34 @@
+use crate::database::{columns, Database, KvStoreError};
+use fuel_storage::Storage;
+use fuel_types::Address;
+use std::{borrow::Cow, collections::HashMap};
+
+impl Storage<u64, HashMap<Address, u64>> for Database {
```

Comment:
Should we maybe use a new-type similar to BlockHeight for this? That way it's more clear from a type-system perspective whether this is the fuel block height vs da block height?

Reply:
I didnt have it inside fuel-interfaces and i defaulted to u64, but yeah changing to BlockHeight is better